Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM docker.adeo.no:5000/pus/node as node-builder
```

Comment:
om du ønsker å kunne bygge denne på egen maskin (kan være greit med tanke på testing), så kan du bruke "args-prefix" opplegget vi har brukte andre steder for å bruke ulikt base-image lokalt og på egen maskin. 

altså:
```
ARG BASE_IMAGE_PREFIX=""
FROM ${BASE_IMAGE_PREFIX}node as builder
```

Da vil prefixen bli satt på byggserveren

Reply:
Skader ikke å ha det på plass. :)